src/Umbraco.Web.UI.Client/src/views/propertyeditors/listview/listview.controller.js

```diff
@@ -642,10 +644,23 @@ function listViewController($scope, $routeParams, $injector, $timeout, currentUs
         }
 
         if (foundAlias) {
+
             value = properties[index].value;
+
+            if (value.startsWith("umb://")) {
+                contentResource.getById(value).then(function (node) {
+                    if (node) {
+                        deferred.resolve(node.variants[0].name);
```

Comment:
Maybe have a look at this PR and see if it's possible to do the same to get the node name with the current culture?

https://github.com/umbraco/Umbraco-CMS/pull/5408

Reply:
Thanks Sebastiaan. The code is different to that PR but I found the way to get the current culture.